src/components/Decks/DeckOfCards.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Deck from './Deck.js';
+
+const DeckOfCards = ({ props }) => {  
+
+  return (
+    <>
+      <Deck title={'Characters'} props={props}/>
```

Comment:
Deck is set up to take "items" instead of "props" - currently your page isn't rendering because of this discrepancy. You can fix it by changing this to:

```<Deck title={'Characters'} items={props} />```

or, you can rename the items to props in Deck.